src/stories/containers/Finances/components/ConditionalWrapper/ConditionalWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from '@emotion/styled';
+import Container from '@ses/components/Container/Container';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+
+interface Props extends React.PropsWithChildren {
+  period: string;
```

Comment:
We should use an enum or a `type` for the periods ('Anually', 'Quarterly', etc) in order to avoid error with the values

Reply:
you are right, done